bin/playwright.sh

```diff
@@ -12,3 +12,5 @@ export TEST_COMMAND=${TEST_COMMAND:-test:playwright:local}
 echo Running Playwright v$PLAYWRIGHT_VERSION as $USER_ID with Playwright arguments $PLAYWRIGHT_ARGS
 
 docker-compose -f docker-compose.playwright.yml up --build --force-recreate --abort-on-container-exit --exit-code-from playwright --remove-orphans
+
+docker-compose -f docker-compose.playwright.yml down
```

Comment:
nice to include this!

Reply:
It's interesting, I thought that the `--abort-on-container-exit` was meant to do this, but I guess it doesn't work as I expected!